docs/mixins/state.md

```diff
@@ -1,41 +1,76 @@
-# Marionette.Toolkit.StateClass
+# StateMixin
 
-`Marionette.Toolkit.StateClass` is a `Marionette.Object` with a `Backbone.Model` for keeping state.
+`StateMixin` is a public mixin for [`App`](./../app.md) or any Marionette class definition that uses a `Backbone.Model` for keeping state.
 
-Use a `StateClass` if your object needs to maintain information that isn't business data.  This provides a consistent method for storing and getting state, along with triggering related events.
+Use a `StateMixin` if your object/view needs to maintain information that isn't business data.  This provides a consistent method for storing and getting state, along with triggering related events.
 
 ## Documentation Index
 
-* [StateClass's `StateModel`](#stateclasss-statemodel)
-* [StateClass's `stateDefaults`](#stateclasss-statedefaults)
-* [StateClass's `stateEvents`](#stateclasss-stateevents)
-* [StateClass API](#stateclass-api)
+* [Using StateMixin](#using-statemixin)
+* [StateMixin's `StateModel`](#statemixins-statemodel)
+* [StateMixin's `stateDefaults`](#statemixins-statedefaults)
+* [StateMixin's `stateEvents`](#statemixins-stateevents)
+* [StateMixin API](#statemixin-api)
   * [Setting State `setState`](#setting-state)
   * [Getting State `getState`](#getting-state)
-  * [Destroying A StateClass `destroy`](#destroying-a-stateclass)
 
-### StateClass's `StateModel`
+### Using StateMixin
 
-Specify a `StateModel` in your state class definition. This must be
+While `StateMixin` comes pre-mixined with `Marionette.Toolkit.App` you can extend your own class with `StateMixin` by calling `initState` in your class's `constructor` passing any desired options.
```

Comment:
change `constructor` to `initialize` end-use classes can just use `initialize`  otherwise they have to call the prototype's constructor or it will break the Object.  We use constructor internally so that `initialize` is still available to the end user.